### DISCUSSION TITLE
Windows unlock error (auto reconnect)

### DISCUSSION THREAD

**Discussion by MrChenhtlss** [rustdesk/rustdesk#6124] [General]
### Bug Description

Remotely control Windows. If you enter a password to unlock the controlled PC when it is on the lock screen, reconnection or connection errors will occur.

### How to Reproduce

Before connecting using RustDesk, log out of controlled Windows in advance.

### Expected Behavior

I hope this issue can be resolved

### Operating system(s) on local side and remote side

MACOS>Windows10

### RustDesk Version(s) on local side and remote side

1.2.4>1.2.4

### Screenshots

![WechatIMG516](https://github.com/rustdesk/rustdesk/assets/146620104/16e0d23a-cad2-4a1c-9f72-9d69d7ef351f)
![WechatIMG517](https://github.com/rustdesk/rustdesk/assets/146620104/9cb33200-3e36-43e2-9ccc-9423b9e2a028)

**Comment by rustdesk** [comment]
@21pages there is supposed to auto reconnect

**Reply by rustdesk** [reply]
it will be more friendly if showing how long will auto reconnect. Current auto reconnect silently, looks wierd

**Reply by rustdesk** [reply]
the dialog does not disappear when auto connect also very wierd.

**Reply by MrChenhtlss** [reply]
Yes, it will automatically reconnect, but it feels weird

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/6125
